### DISCUSSION TITLE
TypeError: Cannot read property 'findMany' of undefined

### DISCUSSION THREAD

**Discussion by LGmatrix13** [prisma/prisma#6407] [Q&A] [answered]
My api route:
```
import prisma from '../../../lib/prisma'

export default async function handle(req, res) {
  const collections = await prisma.collection.findMany({
    take: 1,
  })
  res.json(collections)
}
```
My models:
```
datasource db {
  provider = "sqlite"
  url      = "file:./dev.db"
}

generator client {
  provider = "prisma-client-js"
}

model Post {
  id             String      @id @default(cuid()) 
  image String
  url   String
  user  String
  tag String 
}

model Collection {
  id             String      @id @default(cuid()) 
  tag String
  image   String
}
```
The Error:
`TypeError: Cannot read property 'findMany' of undefined`

I thought I referenced my model correctly in the API route, but I could be wrong. Any suggestions? Thank you in advance!

**Comment by LGmatrix13** [comment, marked as answer]
Must have been a bug with my nextjs environment. I simply killed the local server and re-ran it via `npm run dev`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
